data/Language/Swift.yaml

```diff
@@ -15,6 +15,15 @@ delimiters:
   - round_braces
   - square_braces
   - curly_braces
+datatypes:
+  - bool_swift
```

Comment:
this is the same as many other `bool` datatype definitions.  We only need one.  it should be called bool_lower_case , similar to bool_title_case in https://github.com/coala/coAST/pull/21

And all other lang defs should refer to it if they have the same data type